src/java.base/share/classes/sun/security/tools/keytool/CertAndKeyGen.java

```diff
@@ -82,45 +82,27 @@ public CertAndKeyGen (String keyType, String sigAlg)
     }
 
     /**
-     * Creates a CertAndKeyGen object for a particular key type,
-     * signature algorithm, and provider.
-     *
-     * @param keyType type of key, e.g. "RSA", "DSA"
-     * @param sigAlg name of the signature algorithm, e.g. "MD5WithRSA",
-     *          "MD2WithRSA", "SHAwithDSA". If set to null, a default
-     *          algorithm matching the private key will be chosen after
-     *          the first keypair is generated.
-     * @param providerName name of the provider
-     * @exception NoSuchAlgorithmException on unrecognized algorithms.
-     * @exception NoSuchProviderException on unrecognized providers.
+     * See doc comments at:
+     * constructor CertAndKeyGen(String keyType, String sigAlg,
+     * String providerName, PrivateKey signerPrivateKey,
+     * X500Name signerSubjectName)
```

Comment:
You can simply add a `@see #CertAndKeyGen(String, String, String, PrivateKey, X500Name)`.

Reply:
Done.